remoteworkitem/scheduler_test.go

```diff
@@ -13,29 +13,25 @@ import (
 var db *gorm.DB
 
 func TestMain(m *testing.M) {
-	if _, c := os.LookupEnv(resource.Database); c == false {
-		fmt.Printf(resource.StSkipReasonNotSet+"\n", resource.Database)
-		return
+	if _, c := os.LookupEnv(resource.Database); c != false {
```

Comment:
This is good for now, but in the long term I prefer having the database connection code in one place, and invoked wherever required instead of multiple copies of the same code :)

Something I was trying as part of https://github.com/almighty/almighty-core/pull/112